_distro_map.yml

```diff
@@ -195,16 +195,6 @@ openshift-rosa-portal:
     enterprise-4.13:
       name: ''
       dir: rosa-portal/
-partner-roks:
```

Comment:
@wgordon17, I don't think this distro is in use anymore. Do you have any concerns about removing it?

Reply:
@dfitzmau, we're trying to figure out what's going on with this distro. The [docs](https://docs.openshift.com/roks/4/welcome/index.html) exist, even though they're small and, to my knowledge, unmaintained. For now, please restore the distro section and correct the name instead.

Reply:
Thanks, @kalexand-rh . I restored the original file's content. I'm not sure if I need to change the name as I'm uncertain what part of our doc set this file applies.

Maybe https://docs.openshift.com/roks/4/welcome/index.html but I'm not sure how the file's entries would render to this name unless something add "name+(sub)name+site_name" parameters were bundles together by some backend file.

Reply:
@dfitzmau, my apologies. Will you delete it again?

Reply:
Yes, NP. Removed the section from the `_distro` file.

Reply:
Hi @kalexand-rh . I do not see the doc book for `OpenShift on IBM Cloud` when I navigate to the root directory of the preview link. Maybe removing the entry from the `_distro` YAML file removed this doc book? I have been informed that the title of this doc book is wrong: `Red Hat OpenShift on IBM Cloud 4 Documentation` > `Red Hat OpenShift 4 on IBM Cloud Documentation`. We can drop the `4` also, so the book is a bit more future-proof (if we need to keep the doc book.